client/src/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import styled from 'styled-components';
+
+import Overview from './components/Overview/Overview';
```

Comment:
Are the imports for the component files supposed to have any designation such as '.js' or '.jsx'?

Reply:
We originally had them with the .jsx but the AirBNB style guide said to leave them out!